schema.graphql

```diff
@@ -480,20 +462,12 @@ type Indexer @entity {
   "The total amount of indexing rewards given to delegators"
   delegatorIndexingRewards: BigInt!
 
-  # Capacity data
-  "Amount of delegated tokens that can be eligible for rewards"
-  delegatedCapacity: BigInt!
-  "Total token capacity = delegatedCapacity + stakedTokens"
-  tokenCapacity: BigInt!
-  "Stake available to earn rewards. tokenCapacity - allocationTokens - lockedTokens"
-  availableStake: BigInt!
```

Comment:
I'm quite certain people are using at least this one, should we still remove it with the `calculateCapacities()` function?

If so, we probably need to announce the change with a big disclaimer

Reply:
Agreed, let's put it in the change log for the big release.

The reason I want to remove it is because I don't see a good way that we can keep this consistent. I think it is very possible we change the delegation capacity within the next 6 months, and this will lead many other people to have bad delegation capacities. 

If we can think of a way we could make this work, I would say let's keep it. But I can think of any :(

Reply:
Okay we will keep it in. We introduce some more fields in a newer PR that also has the same issue.  So might as well keep them both in for now. Plan is to make note of the issue in the repository, and figure out a way in the future to avoid this issue, if possible